docs/types/value-types.rst

```diff
@@ -157,23 +157,62 @@ Fixed Point Numbers
 -------------------
 
 .. warning::
-    Fixed point numbers are not fully supported by Solidity yet. They can be declared, but
-    cannot be assigned to or from.
+    Fixed point numbers are not fully supported by Solidity yet.
 
-``fixed`` / ``ufixed``: Signed and unsigned fixed point number of various sizes. Keywords ``ufixedMxN`` and ``fixedMxN``, where ``M`` represents the number of bits taken by
-the type and ``N`` represents how many decimal points are available. ``M`` must be divisible by 8 and goes from 8 to 256 bits. ``N`` must be between 0 and 80, inclusive.
+``fixed`` / ``ufixed``: Signed and unsigned fixed point number of various sizes. Keywords ``ufixedMxN`` and ``fixedMxN``, where ``M`` represents the total number of bits used by
+the type and ``N`` represents how many fractional decimal digits are available. ``M`` must be divisible by 8 and goes from 8 to 256 bits. ``N`` must be between 0 and 80, inclusive.
```

Comment:
```suggestion
the type and ``N`` represents how many fractional decimal digits are available. The number ``M`` must be divisible by 8 and goes from 8 to 256 bits and ``N`` must be between 0 and 80, inclusive.
```

Reply:
I wonder if we need to include `0` for `N`.

Reply:
It might be useful in the future, because it is clear that the result of `fixed128x2 * fixed64x0` is a fixed point number, while it is not clear for `fixed128x2 * int64`.

But we can also remove it now and re-introduce it when we think it is useful.